tael.md

```diff
@@ -52,6 +52,33 @@ This should trigger the client you set up in the previous step (as well as print
 
 If your topic includes the substring `test` it will be supressed from `#tael-stream` (the client will announce that it has received a message on the topic `COGUK/infrastructure/pipelines/<CLIENT_NAME>/status` however).
 
+Running pipelines have a control topic `COGUK/infrastructure/pipelines/<PIPELINE>/control` which can be used to control the pipeline (e.g. re-raise it if it has failed), however this should not be accessed by anyone not directly responsible for the running of a pipeline.
```

Comment:
remove "however this should not be accessed by anyone not directly responsible for the running of a pipeline."